ruby/basic_ruby/nested_collections.md

```diff
@@ -320,32 +330,34 @@ vehicles.filter_map { |name, data| name if data[:year] >= 2020 }
 Amazing! We have found a great solution to returning an array that only contains the names of the owners of vehicles from 2020 or newer! Plus, we got experience using other methods that you will probably use in the future. We have found some really useful methods by exploring the documentation when we have a specific use case in mind.
 
 ### Assignment
+
 <div class="lesson-content__panel" markdown="1">
 
-1. To learn more about using a hash with a nested array, read [this article](https://web.archive.org/web/20220525215038/https://learn.co/lessons/nested-hash-iteration) from Learn.co.
-2. To learn more about using an array with a nested hash, read [the answer to this post](https://stackoverflow.com/questions/50529389/manipulating-output-from-an-array-of-nested-hashes-in-ruby) on Stack Overflow.
-3. Complete the [nested collections](https://github.com/TheOdinProject/ruby-exercises/tree/main/ruby_basics) exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises) that you previously cloned.
+1. Read more about [using a hash with a nested array](https://web.archive.org/web/20220525215038/https://learn.co/lessons/nested-hash-iteration).
+1. This Stack Overflow answer explains more about [using arrays with a nested hash](https://stackoverflow.com/questions/50529389/manipulating-output-from-an-array-of-nested-hashes-in-ruby).
+1. Complete the nested collections exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises)
```

Comment:
A definite nit, but we should be consistent with punctuation.
```suggestion
1. Complete the nested collections exercises from the [ruby-exercises repo](https://github.com/TheOdinProject/ruby-exercises).
```

Reply:
Done!